charmcraft.yaml

```diff
@@ -31,3 +31,34 @@ parts:
     plugin: uv
     build-snaps:
       - astral-uv
+  falco:
+    plugin: nil
+    override-build: |
```

Comment:
How much time does it take to build on the fly? The problem I see with this approach is that release of the falco tarball and the charm have a different lifecycle despite being in the same repository. Building the sources as part of the pack would solve this (if it's feasible)